### ISSUE TITLE
Update Microscope to use iron-router

### ISSUE THREAD

**Issue by DominikGuzei** [DiscoverMeteor/book#127]
Hey there,

First I want to thank you for your awesome book about Meteor!
Its definitely the best resource while learning the framework so far ;-)

One thing that could be improved (updated) is the usage of the old "router" package. If you go to the Atmosphere website, this packages is stated as deprecated in favor of "iron-router" which is much more polished.

Although it wasn't hard to follow along with iron-router, it definitely would be better to show other newbies the best package for routing right from the start!

Keep up the good work,
Dominik

**Comment by SachaG** [comment]
Glad you liked the book! And don't worry, we're working on it :)
